drivers/clock_control/clock_control_nrf2_hfxo.c

```diff
@@ -29,6 +32,31 @@ struct dev_config_hfxo {
 	k_timeout_t start_up_time;
 };
 
+#if defined(CONFIG_ZERO_LATENCY_IRQS)
+static uint32_t full_irq_lock(void)
+{
+	uint32_t mcu_critical_state;
+
+	if (IS_ENABLED(CONFIG_ZERO_LATENCY_IRQS)) {
```

Comment:
This is redundant? Whole function is #ifdef'd in line 35.

Reply:
Fixed in https://github.com/zephyrproject-rtos/zephyr/pull/82995